intermediate-js.js

```diff
@@ -18,23 +22,70 @@ var bicycle = {
 }
 
 // Log the type of bicycle:
-console.log()
+console.log(bicycle.type)
+//Done
 
 // Log the bell:
-console.log()
+console.log(bicycle.gear[2])
+//Done
 
 // Log the PSI:
-console.log()
+console.log(bicycle.wheels.specs[2])
+//Done
 
 
 // 3. Write a function called letterCounter that logs the number of times the letter "l" appears in ourString.
 
 var ourString = "Hello Learn Students!"
 
+countLetter = (str, letter) => {
+	let amount = 0
+	str = str.toLocaleLowerCase()
+	for(let i = 0; i<str.length;i++){
+		if(str.charAt(i) === letter){
+			amount = amount+1 
+		}
+	}
+	return amount
+}
+console.log(countLetter(ourString, 'l'))
+//Done
 
 // 4. Write a function called getFib that returns the first 10 numbers of the Fibonacci sequence in an array. [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+getFib = (num) => {
+	var newFib = [0,1]
+	for(let i = 2; i<11; i++){
+		newFib[i] = newFib[i -2] + newFib[i -1]
+	}
+	return newFib
+}
+console.log(getFib(10))
+//Done
 
```

Comment:
The argument you are passing isn't acting in your code. To make this more dynamic consider altering your loop length.